lib/eex_html.ex

```diff
@@ -136,7 +136,7 @@ defmodule EExHTML do
   @doc """
   This module adds `~E` sigil for safe HTML escaped content.
   """
-  defmacro sigil_E({:<<>>, [line: line], [template]}, []) do
+  defmacro sigil_E({:<<>>, [indentation: _, line: line], [template]}, []) do
```

Comment:
It would be best to not match on the metadata. Instead do:

```suggestion
  defmacro sigil_E({:<<>>, meta, [template]}, []) do
```

and then do `line: meta[:line] + 1` below.

Reply:
Thanks for reviewing this.  I made the changes you suggested and I think they are correct.